src/supervisor3.app.src

```diff
@@ -1,14 +1,13 @@
 %% -*- mode:erlang -*-
 {application,supervisor3,
  [{description,"A copy of supervisor.erl from the R16B Erlang/OTP with modifications"},
-  {vsn,"1.1.8"},
+  {vsn,"1.1.9"},
```

Comment:
Dropping build systems and OTP releases is a breaking change, so according to semver I should've created a `2.0.0` version instead. But given that this pr makes zero functional changes, I am not so sure.

Reply:
I think 1.2.0 would be appropriate.